src/main/java/com/maschel/roomba/RoombaJSSC.java

```diff
@@ -376,8 +376,8 @@ public void drivePWM(int rightPWM, int leftPWM) throws IllegalArgumentException
             throw new IllegalArgumentException("PWM should be between -100% and 100%");
 
         log.info("Sending 'drivePWM' command (right PWM: " + rightPWM + "%, left PWM: " + leftPWM + "%) to roomba.");
-        int relRightPWM = (DRIVE_WHEEL_MAX_POWER * rightPWM/100);
-        int relLeftPWM = (DRIVE_WHEEL_MAX_POWER * leftPWM/100);
+        int relRightPWM = (int) (DRIVE_WHEEL_MAX_POWER * (rightPWM / 100.0));
```

Comment:
**What are you fixing here?**

The old code delivers the same result as your code, only your code adds unneccesary casting:
1. rightPWM will cast to double (because of the 100.0)
2. DRIVE_WHEEL_MAX_POWER will cast to double (because of the double result after division inside the    parentheses)
3. The result of the statement (double) is casted back to int

Please see the output of this program:
![screen shot 2017-10-18 at 11 55 29](https://user-images.githubusercontent.com/1552923/31713091-6c1c7ed6-b3fd-11e7-8f8e-be3d6ba2c6c6.png)